src/Event/IndexEvent.php

```diff
@@ -11,7 +11,7 @@
 
 namespace FOS\ElasticaBundle\Event;
 
-use Symfony\Component\EventDispatcher\Event;
+use Symfony\Contracts\EventDispatcher\Event;
```

Comment:
Older Symfony versions might not have the contracts component.

Reply:
You could create [a `BCEvent` and extend from it](https://github.com/liip/LiipImagineBundle/pull/1218).